src/deploy/functions/runtimes/node/parseTriggers.ts

```diff
@@ -57,6 +57,7 @@ export interface TriggerAnnotation {
   timeZone?: string;
   regions?: string[];
   concurrency?: number;
+  invoker?: string | string[];
```

Comment:
Let's make this an array always. Your code will be cleaner later.

Reply:
Changed